store/errors.go

```diff
@@ -163,12 +161,8 @@ func SanitizeError(err error) error {
 		return nil
 	}
 	var errHTTP = ErrHTTPOpFailed
-
-	switch {
-	case errors.Is(err, ErrItemNotFound):
+	if errors.Is(err, ErrItemNotFound) {
```

Comment:
Codecov annotated here.  I looked and it looks like we don't have any `errors_test.go` file, which means no unit tests for the code in this file? This seems tangential to this pr so I propose we make an issue and fix this later.

Reply:
Sounds good 👍  Added https://github.com/xmidt-org/argus/issues/172